libs/subliminal_patch/providers/subssabbz.py

```diff
@@ -144,8 +154,17 @@ def query(self, language, video):
                     link = element.get('href')
                     element = row.find('a', href = re.compile(r'.*showuser=.*'))
                     uploader = element.get_text() if element else None
-                    logger.info('Found subtitle link %r', link)
-                    sub = self.download_archive_and_add_subtitle_files(link, language, video)
+                    cells = row.findChildren('td')
+
+                    sub_disks = int(cells[6].getText()) if cells else 1
+                    if sub_disks != 1:
+                        logger.info('Multi-disk subtitles (unsupported). Skipping %r', link)
+                        continue
```

Comment:
Open https://subsunacs.net/search.php?m=alien%20vs%20predator&y=2004 and click Search - some results are for a single video file, others are for multiple. If you download any of the multi-file subtitles, you will end up with half subtitled movie, which doesn't make sense as a viable result at all.

Remember the purpose here - we want to rule out all false results that do not give you viable subtitles, plus score up the ones that have accurate fps and score down the ones that have mismatching fps.

Reply:
Ok with me.

Reply:
I would accept removal of search results only if the name of the movie or year is wrong. And in case of the tv show - wrong series name, season or episode.